### PR TITLE
Update 0021-django-admin.md

### PR DESCRIPTION
**Pull request by rachidatecs**
## 🗣 Description ##

- Update with notes on accessibility and theming

## 💭 Motivation and context ##

This revises the ADR to include subsequent findings.

### CODE REVIEW

File: docs/architecture/decisions/0021-django-admin.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ In contrast to building an admin interface from scratch where development activi
 involve _building up_, leveraging Django Admin will require carefully _pairing back_ the functionalities available to
 users such as analysts.
 
+On accessibility: Django admin is almost fully accessible out-of-the-box, the expections being tables, checkboxes, and
```
Comment (neilmb) [inline docs/architecture/decisions/0021-django-admin.md:37]:
```suggestion
On accessibility: Django admin is almost fully accessible out-of-the-box, the exceptions being tables, checkboxes, and
```

**Review by neilmb — COMMENTED** [review]
LGTM, thanks for the additional details

File: docs/architecture/decisions/0021-django-admin.md (hunk 1 above)
Comment (abroddrick) [reply docs/architecture/decisions/0021-django-admin.md:37]: Finally Rachid has a spelling error and not just me!

**Review by abroddrick — APPROVED** [review]
Looks good, but I before approving see the minor change Neil suggests!

File: docs/architecture/decisions/0021-django-admin.md (hunk 1 above)
Comment (rachidatecs) [reply docs/architecture/decisions/0021-django-admin.md:37]: The expectorations?

**Pull request merged**